lib/OpenQA/CLI/api.pm

```diff
@@ -99,9 +99,14 @@ sub command ($self, @args) {
     openqa-cli api -X POST jobs ISO=foo.iso DISTRI=my-distri \
       FLAVOR=my-flavor VERSION=42 BUILD=42 TEST=my-test
 
-    # Trigger jobs on ISO "foo.iso"
-    openqa-cli api --o3 -X POST isos ISO=foo.iso DISTRI=my-distri \
-      FLAVOR=my-flavor ARCH=my-arch VERSION=42 BUILD=1234
+    # Trigger jobs on ISO "foo.iso" creating a "scheduled product" (see
+    # https://open.qa/docs/#_spawning_multiple_jobs_based_on_templates_isos_post
+    # for details)
+    openqa-cli api --o3 -X POST isos async=0 ISO=foo.iso \
```

Comment:
The comment says "scheduled product" but then you use again async=0? Did you want to use async=1 here? I guess you should say `async=1` here to show this "advanced" cases whereas on top in lines 99-100 the simple, default cases is shown

Reply:
No, the "scheduled product" is created in any case. I've just mentioned the term here so it is established to be used for the next example. (You can also check the state of scheduled products created with `async=0` so there's also no conflict with that either.)

Reply:
Removed the parameter.